### PR TITLE
Allow <1 dashdot values

### PR DESCRIPTION
**Pull request by pastk**

### CODE REVIEW

File: src/libkomwm.py [hunk 1]
```diff
@@ -722,12 +722,8 @@ def rule_sort_key(dict_):
...
                                 dr_line.priority = get_drape_priority(cl, 'line', st.get('object-id'))
                                 store_visibility(cl, 'line', st.get('object-id'), zoom)
-                            dd = st.get('casing-dashes', st.get('dashes', []))
-                            for i in dd:
-                                if float(i) < 1:
-                                    print(f'WARNING: forcing minimum dashdot value to 1 in "{dd}" for z{zoom} line(casing) {cl}')
-                                    i = 1.0
-                                dr_line.dashdot.dd.extend([i * WIDTH_SCALE])
+                            for i in st.get('casing-dashes', st.get('dashes', [])):
+                                dr_line.dashdot.dd.extend([float(i)])
```
Comment (vng) [inline src/libkomwm.py:724]: WIDTH_SCALE not needed ?

Comment (vng) [reply src/libkomwm.py:724]: UPD: WIDTH_SCALE == 1. I'd keep it for consistency with other values.

Comment (pastk) [reply src/libkomwm.py:724]: Effectively `WIDTH_SCALE` is not used anywhere (an looks like never was used), so I see no point in keeping it in the code.

I can remove all usages of it (its just a few) at once in this PR for consistency. WDYT?

Comment (vng) [reply src/libkomwm.py:724]: Remove if you think it will not be used.

Comment (pastk) [reply src/libkomwm.py:724]: Done

**Comment by vng** [comment]
Did you generate? No problem with int-float conversion?

**Comment by pastk** [comment]
Yes I regenerated and there are no conversion problems.

**Pull request merged**